packages/opentelemetry-tracing/src/Span.ts

```diff
@@ -95,13 +95,37 @@ export class Span implements types.Span, ReadableSpan {
     return this;
   }
 
-  addEvent(name: string, attributes?: types.Attributes): this {
+  /**
+   *
+   * @param name Span Name
+   * @param [attributes] Span attributes or start time
+   * if type is {@type TimeInput} and 3rd param is undefined
```

Comment:
Nit: I usually recommend a 4 space indent for continued JSDoc lines so that it's clearer that its part of the previous entry.